_targets/build.project.armv7a9-zynq7000

```diff
@@ -8,13 +8,32 @@
 [ "${BASH_SOURCE[0]}" -ef "$0" ] && echo "You should source this script, not execute it!" && exit 1
 
 
-b_build_target() {
-	echo "b_build_target"
+# Pre-init script is launched before user script
+PREINIT_SCRIPT=("map ddr 100000 900000 rwx"
+                "phfs usb0 1.2 phoenixd"
+                "phfs uart0 0.0 phoenixd"
+                "phfs uart1 0.1 raw"
+                "console 0.1"
+                "wait 1500")
+
+
+b_mkscript_preinit() {
+	for cmd in "${PREINIT_SCRIPT[@]}"; do
+		printf "%s\n" "$cmd" >> "$PREFIX_BUILD/plo/script.plo"
+	done
+}
+
 
-	cat _targets/armv7a9-zynq7000.plo >> "$PREFIX_BUILD/plo/script.plo"
-	printf "\0" >> "$PREFIX_BUILD/plo/script.plo"
+b_build_target() {
+	b_log "Building sample project for $TARGET"
 
+	# FIXME: build plo out of tree and without clean
 	b_log "Building plo"
+	make -C plo clean all
+
+	b_mkscript_preinit
```

Comment:
why can't we write script before `plo` and compile just once?

Reply:
Removed. There is only one compilation.